### DISCUSSION TITLE
Two equations used for tau approximation

### DISCUSSION THREAD

**Discussion by danielee0707** [theislab/scvelo#476] [Q&A]
Thank you for the wonderful method! I have a question about the tau approximation part in the dynamical model. Why are there two separate formulas used for beta > gamma and beta <= gamma? One uses the unspliced equation while the other one uses the spliced equation. What's the difference between these two? Thanks!

**Comment by NajwaLaabid** [comment]
It comes from the "explicit formula for approximating optimal time assignment". The text below is from the [published paper](https://www.nature.com/articles/s41587-020-0591-3?proof=t), under Methods -> Parameter inference:


<img width="638" alt="Screen Shot 2020-09-18 at 11 13 22" src="https://user-images.githubusercontent.com/17838639/93573882-5d159300-f9a0-11ea-8491-c3e2e177aed9.png">



My understanding is that the "beta < gamma" condition makes equation (10) possible (it's a condition for the formula inside the log() to be positive). I tried deriving the formula manually but I couldn't reach an explicit expression showcasing this dependence, not without considering s(inf) necessarily smaller than s(0) and s(t) in all cases (which doesn't seem unreasonable from my very basic biological knowledge). In the case of beta >= gamma, we can't use equation (10) so we use (11) as an alternative (formula derivable from the equation of u(t) given in (4), same paper).

I am not sure my reasoning on (10) is correct. Also not sure why we cannot use (11) as the only method for approximation for all genes. Maybe someone can take it from here and explain these two points :)

**Comment by VolkerBergen** [comment]
The rationale is simply that beta > gamma guarantee that we're tilde beta> 0, thus is given by a linear combination s - tilde beta u, where u contributes with a negative factor, which then makes sure that the explicit time assignment approximates an optimal orthogonal projection onto the phase trajectory. If beta < gamma, both would have a positive factor, such that a direct inversion of u is a better proxy than combining s and u both with positive factors. Hope the fig sheds some light (here: c>0). 

Biologically, we would mostly expect beta > gamma. 

![image](https://user-images.githubusercontent.com/31883718/95198984-65613100-07dc-11eb-973e-863d71e0bbc0.png)

**Comment by danielee0707** [comment]
Hi Volker, just want to ask a follow-up question. I think when beta > gamma, the tilda beta is actually negative, isn't it? That makes the red line the approximation in the case of s - tilda beta u.

**Reply by WeilerP** [reply]
@danielee0707, yes you are right, the shown figure is not correct - it should be the other way around. You can also show this analytically.
